ts/screens/services/ServicesWebviewScreen.tsx

```diff
@@ -0,0 +1,88 @@
+import CookieManager, { Cookie } from "@react-native-community/cookies";
+import { fromNullable } from "fp-ts/lib/Option";
+import { Content } from "native-base";
+import * as React from "react";
+import { SafeAreaView, StyleSheet } from "react-native";
+import { NavigationEvents } from "react-navigation";
+import { connect } from "react-redux";
+import URLParse from "url-parse";
+import RegionServiceWebView from "../../components/RegionServiceWebView";
+import BaseScreenComponent from "../../components/screens/BaseScreenComponent";
+import { navigateBack } from "../../store/actions/navigation";
+import { Dispatch } from "../../store/actions/types";
+import { tokenFromSessionInfoSelector } from "../../store/reducers/authentication";
+import { internalRouteNavigationParamsSelector } from "../../store/reducers/internalRouteNavigation";
+import { GlobalState } from "../../store/reducers/types";
+
+type Props = ReturnType<typeof mapStateToProps> &
+  ReturnType<typeof mapDispatchToProps>;
+
+const styles = StyleSheet.create({
+  flex: { flex: 1 }
+});
+
+const ServicesWebviewScreen: React.FunctionComponent<Props> = (
+  props: Props
+) => {
+  const [isCookieAvailable, setIsCookieAvailable] = React.useState(false);
+  const [cookieError, setCookieError] = React.useState(false);
+
+  const onDidFocus = () => {
```

Comment:
I did few test and this callback is never called (maybe a bug on navigation library)
Perhaps you could use `React.useEffect` with a clean-up callback. I tested it and it works